tests/unit/test_feed_stored_xss.py

```
@@ -0,0 +1,45 @@
+import pytest
+
+from backend.app import app
+from backend.extensions import db
+from backend.models import Feed, Tab
+
+
+def test_add_feed_invalid_url_scheme_prevented(client, mocker):
```

Ruff (0.15.2)

[warning] Unused function argument: `mocker`

(ARG001)